src/server.js

```diff
@@ -76,7 +76,7 @@ const init = async () => {
 		}
 
 		if (!validPassword(password)) {
-			const message = 'User account password was not valid'
+			const message = 'User account password was not valid because it\'s length is either below 5 or above 200.'
```

Comment:
Can you change the text to say this instead?

Account password length must be between 5 and 200 characters.